version.go

```diff
@@ -2,7 +2,7 @@ package meilisearch
 
 import "fmt"
 
-const VERSION = "0.24.0"
+const VERSION = "0.24.1"
```

Comment:
Since they are new features you can safely use the next minor :)

```suggestion
const VERSION = "0.25.0"
```